src/libstore/build.cc

```diff
@@ -1842,18 +1842,20 @@ void DerivationGoal::startBuilder()
        work properly.  Purity checking for fixed-output derivations
        is somewhat pointless anyway. */
     {
-        string x = settings.get("build-use-chroot", string("false"));
+        string x = settings.get("build-use-sandbox",
+            /* deprecated alias */
+            settings.get("build-use-chroot", string("false")));
         if (x != "true" && x != "false" && x != "relaxed")
-            throw Error("option ‘build-use-chroot’ must be set to one of ‘true’, ‘false’ or ‘relaxed’");
+            throw Error("option ‘build-use-sandbox’ must be set to one of ‘true’, ‘false’ or ‘relaxed’");
         if (x == "true") {
-            if (get(drv->env, "__noChroot") == "1")
-                throw Error(format("derivation ‘%1%’ has ‘__noChroot’ set, but that's not allowed when ‘build-use-chroot’ is ‘true’") % drvPath);
+            if (get(drv->env, "__noSandbox") == "1")
+                throw Error(format("derivation ‘%1%’ has ‘__noSandbox’ set, but that's not allowed when ‘build-use-sandbox’ is ‘true’") % drvPath);
```

Comment:
Renaming `__noChroot` to `__noSandbox` will break packages that rely on this flag. (This shouldn't affect Nixpkgs, but for instance at LogicBlox we have some builds that need this.)

Given the obscurity of this flag, I think it's fine to leave it unrenamed.